int8 ACTION_READ=1
int8 ACTION_WRITE=2
string relay_label
int8 action
int8 write_state
---
int8 relay_state
bool success
